db/migrate/20230301081858_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
- [ ] You do not have a `db/schema.rb` file. Kindly note that after creating these migrations you are still required to run `rails db:migrate` to generate the database schema. If you do that successfully there should be a `db/schema.rb` containing your database structure. Kindly do that.

  ![image](https://user-images.githubusercontent.com/36542986/222106873-8239cafe-3cec-4b43-a3cf-4fb3748daf94.png)

Reply:
thanks for your details inforamtion , i am gonna try it now.